app/assets/v2/js/pages/bulk_payout.js

```diff
@@ -162,7 +162,7 @@ $(document).ready(function($) {
       <tr>
         <td class="pl-0 pb-0">
           <div class="pl-0">
-            <select id="username" onchange="update_registry()" class="username-search custom-select" style="width: 100%; margin-left: auto; margin-right: auto;"></select>
+            <select onchange="update_registry()" class="username-search custom-select" style="width: 100%; margin-left: auto; margin-right: auto;"></select>
```

Comment:
Add `w-100 ml-auto mr-auto` to the class attribute and we can get rid of the `style=...` attribute. :)